python/Ganga/Core/GangaRepository/GangaRepositoryXML.py

```diff
@@ -225,7 +228,10 @@ def startup(self):
             self.from_file = pickle_from_file
         else:
             raise RepositoryError(self.repo, "Unknown Repository type: %s" % self.registry.type)
-        self.sessionlock = SessionLockManager(self, self.lockroot, self.registry.name)
+        if getConfig('Configuration')['lockingStrategy'] == "UNIX":
+            self.sessionlock = SessionLockManager(self, self.lockroot, self.registry.name)
+        else:
```

Comment:
Should check for the value "Fixed" rather than just an elese statement (and any other message should lead to a complete fail).